sharktank/sharktank/utils/testing.py

```diff
@@ -36,8 +36,12 @@
     reason="Run quick tests if --run-quick-test is passed",
 )
 is_nightly = pytest.mark.skipif(
-    'not config.getoption("run-nightly-tests")',
-    reason="Run large tests if --run-nightly-tests is passed",
+    'not config.getoption("run-nightly-test")',
+    reason="Run large tests if --run-nightly-test is passed",
```

Comment:
Actually all nightly tests run whatever is marked [expensive](https://github.com/nod-ai/shark-ai/blob/3f196d60c33dec590dca29aa2ce120b629a5b3b5/sharktank/pyproject.toml#L53). At some point we need to reconcile the various parallel approaches for marking tests for nightly. We should use markers as then you could construct logical expressions to make your test selection.